### PR TITLE
fix: write campuses back to Rock correctly

### PR DESCRIPTION
**Pull request by redreceipt**

### CODE REVIEW

File: packages/apollos-data-connector-rock/src/campuses/data-source.js [hunk 1]
```diff
@@ -114,9 +115,9 @@ export default class Campus extends RockApolloDataSource {
     ).first();
 
     if (!personGroup) return null;
-    const { id: rockCampusId } = parseGlobalId(campusId);
-
-    await this.patch(`/Groups/${personGroup.id}`, { CampusId: rockCampusId });
+    await this.patch(`/Groups/${personGroup.id}`, {
+      CampusId: apollosId ? parseGlobalId(apollosId).id : rockId,
```
Comment (redreceipt) [inline packages/apollos-data-connector-rock/src/campuses/data-source.js:119]: actually @vinnyjth look closely at this, not sure this is right. what does `parseGlobalId(apollosId).id` give us?

**Review by vinnyjth — APPROVED** [review]

File: packages/apollos-data-connector-rock/src/campuses/data-source.js (hunk 1 above)
Comment (vinnyjth) [reply packages/apollos-data-connector-rock/src/campuses/data-source.js:119]: A rock campus id (assuming you don't pass a Postgres Id)

Comment (redreceipt) [reply packages/apollos-data-connector-rock/src/campuses/data-source.js]: there i simplified the signature a bit to make it clearer

Comment (redreceipt) [reply packages/apollos-data-connector-rock/src/campuses/data-source.js]: confirm you like the new params before i merge

**Pull request merged**